access/src/main/external/static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -134,8 +134,13 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities',  'E
 			}
 		}
 		
+		if ($.inArray('changeResourceType', metadata.permissions) != -1
+				&& $.inArray('info:fedora/cdr-model:Container', metadata.model) != -1) {
+			items["changeType"] = {name : 'Change Type'};
```

Comment:
Menu items that modify the object should be grouped together. We could have something like this:

```
--
Unpublish
Edit Access
Edit Label
Edit Description
Change Type
--
Export as CSV
Copy PID to Clipboard
--
```